projects/dashboards/ecometrics/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,540 @@
+                                            },
+                                        ],
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            symbology: new RasterSymbology(
+                1.0,
+                new SingleBandRasterColorizer(
+                    0,
+                    new LinearGradient(
+                        ColorMapSelectorComponent.createLinearBreakpoints(ALL_COLORMAPS.VIRIDIS, 16, false, {min: -1, max: 1}),
+                        // [
+                        // (new ColorBreakpoint(-1, WHITE), new ColorBreakpoint(1, Color.fromRgbaLike([0, 255, 0, 1])))
+                        // ],
+                        TRANSPARENT,
+                        TRANSPARENT,
+                        TRANSPARENT,
+                    ),
+                ),
+            ),
+            dataRange: {min: 0, max: 1},
+            measurement: 'continuous',
+        },
+    ];
+
+    selectedIndicator: Indicator | undefined;
+    selectedBBox: Extent | undefined;
+
+    @ViewChild(MapContainerComponent, {static: false}) mapComponent!: MapContainerComponent;
+    // @ViewChild(TimeStepSelectorComponent, {static: false}) timeSelectorComponent!: TimeStepSelectorComponent;
+
+    @ViewChild('welcome') welcome!: TemplateRef<unknown>;
+    @ViewChild('inspect') inspect!: TemplateRef<unknown>;
+    @ViewChild('select') select!: TemplateRef<unknown>;
+    @ViewChild('review') review!: TemplateRef<unknown>;
+
+    private _viewContainerRef = inject(ViewContainerRef);
+
+    cards = new BehaviorSubject<Array<{title: string; cols: number; rows: number; content: TemplatePortal}>>([]);
+    readonly layersReverse$: Observable<Array<Layer>>;
+
+    timeSteps: Time[] = [
+        new Time(utc('2022-01-01')),
+        new Time(utc('2023-01-01')),
+        // new Time(utc('2024-02-01')),
+        // new Time(utc('2024-03-01')),
+        // new Time(utc('2024-04-01')),
+        // new Time(utc('2024-05-01')),
+        // new Time(utc('2024-06-01')),
+    ];
+
+    readonly plotData$ = new BehaviorSubject<any>(undefined);
+    readonly plotLoading$ = new BehaviorSubject(false);
+
+    constructor(
+        readonly userService: UserService,
+        readonly projectService: ProjectService,
+        readonly dataSelectionService: DataSelectionService,
+        readonly notificationService: NotificationService,
+        readonly mapService: MapService,
+        readonly datasetService: DatasetService,
+        readonly changeDetectorRef: ChangeDetectorRef,
+        readonly backend: BackendService,
+        readonly spatialReferenceService: SpatialReferenceService,
+    ) {
+        this.layersReverse$ = this.dataSelectionService.layers;
+    }
+
+    async ngAfterViewInit(): Promise<void> {
+        this.breakpointObserver
+            .observe(Breakpoints.Handset)
+            .pipe(
```

Comment:
![grafik](https://github.com/user-attachments/assets/602744a5-77aa-4551-953d-184cdbbb2167)

For smaller screens, I cannot select something or read the text

Reply:
true. should I maybe not overlay the legend on the map? or should I add a toggle legends button?

Reply:
Both solutions sound good. Maybe use the one you like best. Or maybe your first one is better for smaller screens :thinking: .

Reply:
Detaching it from the map makes sense, but we should do that when we change the grid layout. For now, I made the legend toggleable

![image](https://github.com/user-attachments/assets/29127b89-82fa-4d4b-ad93-66361c9ebd3f)
![image](https://github.com/user-attachments/assets/3cf6ef1e-810c-4293-b6fb-146451f69097)